go_sequencer/database/statedb/vouch_service.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 var (
```

Comment:
We'll need to add a function for finding the vouchIdx, Since that'll be formed by joining Account FromIdx and ToIdx

You can add that here - [Ref](https://github.com/tokamak-network/tokamak-sybil-resistance/pull/31/files#diff-131e26b3a77b8a4ac3a11c29b67a2ddff470a434e2c846dc4e0a2e4e8786aaa6R114)

Reply:
I've added the func to generate vouchIdx from FromIdx and ToIdx in common/vouch.go
And the things we need to consider is that the length of AccountIdx is 3 bytes.
So the bit shift operation should be by 24(<- 64).

Thanks for your ref.